tests/subtractions/test_db.py

```diff
@@ -2,60 +2,87 @@
 
 import virtool.subtractions.db
 from virtool.data.transforms import apply_transforms
+from virtool.fake.next import DataFaker
 from virtool.subtractions.db import (
     AttachSubtractionsTransform,
     unlink_default_subtractions,
 )
+from virtool.uploads.models import UploadType
 
 
 @pytest.mark.parametrize(
-    "documents",
-    [
-        {"id": "sub_1", "subtractions": ["foo", "bar"]},
-        [
-            {"id": "sub_1", "subtractions": ["foo", "bar"]},
-            {"id": "sub_2", "subtractions": ["foo"]},
-            {"id": "sub_3", "subtractions": []},
-        ],
-    ],
+    "case",
+    ["single", "multiple"],
```

Comment:
I see what you were saying about this test.

I think the most elegant solution is to convert to a class-based test with `test_single` and `test_multi` methods.

Reply:
With the parameterization it has now it runs in parallel when using`xdist`

Reply:
It should either way.